skorch/net.py

```diff
@@ -1377,6 +1377,22 @@ def check_is_fitted(self, attributes=None, *args, **kwargs):
         )
         check_is_fitted(self, attributes, *args, **kwargs)
 
+    def __sklearn_is_fitted__(self):
+        """This method is called when sklearn's ``check_is_fitted`` is used.
+        
+        Explained here: 
+        https://scikit-learn.org/stable/auto_examples/developing_estimators/sklearn_is_fitted.html
+        """
+        from skorch.exceptions import NotInitializedError
+        try:
+            self.check_is_fitted()
+        except NotInitializedError:
+            from sklearn.exceptions import NotFittedError
```

Comment:
Thanks for updating the method. You can make these imports global, it's only in the tests that we use local imports.